jetstream/probe_sets.py

```diff
@@ -46,9 +44,13 @@ def columns_for_scalar(self, scalar_name: str) -> List[str]:
 
 
 @attr.s(auto_attribs=True, kw_only=True, slots=True, frozen=True)
-class FeatureScalarTelemetry:
+class ScalarProbe:
     kind: ClassVar[str] = "scalar"
     name: str
+    event_category: str
+    event_method: Optional[str] = None
+    event_object: Optional[str] = None
+    event_value: Optional[str] = None
```

Comment:
Why would we add event fields to a scalar type?

Reply:
copy-paste